modules/ROOT/pages/browser-support.adoc

```diff
@@ -0,0 +1,26 @@
+= Browser Support for Anypoint Platform
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+// page-aliases: 
```

Comment:
This line is commented out for now--other writers will be adding page aliases in the next sprint. So this line will change before being merged.